core/chains/evm/logpoller/orm.go

```diff
@@ -299,59 +326,68 @@ func (o *DbORM) DeleteExpiredLogs(limit int64, qopts ...pg.QOpt) (int64, error)
 			) r ON l.evm_chain_id = $1 AND l.address = r.address AND l.event_sig = r.event
 			AND l.block_timestamp <= STATEMENT_TIMESTAMP() - (r.retention / 10^9 * interval '1 second')
 			LIMIT $2
-		)`,
-			ubig.New(o.chainID), limit)
-	}
-
-	return q.ExecQWithRowsAffected(`WITH r AS
+		)`, ubig.New(o.chainID), limit)
+	} else {
+		result, err = o.db.ExecContext(ctx, `WITH r AS
 		( SELECT address, event, MAX(retention) AS retention
 			FROM evm.log_poller_filters WHERE evm_chain_id=$1 
 			GROUP BY evm_chain_id,address, event HAVING NOT 0 = ANY(ARRAY_AGG(retention))
 		) DELETE FROM evm.logs l USING r
 			WHERE l.evm_chain_id = $1 AND l.address=r.address AND l.event_sig=r.event
-			AND l.block_timestamp <= STATEMENT_TIMESTAMP() - (r.retention / 10^9 * interval '1 second')`, // retention is in nanoseconds (time.Duration aka BIGINT)
-		ubig.New(o.chainID))
+			AND l.created_at <= STATEMENT_TIMESTAMP() - (r.retention / 10^9 * interval '1 second')`, // retention is in nanoseconds (time.Duration aka BIGINT)
```

Comment:
What is the difference between `block_timetamp` and `created_at`

Reply:
Should be block_timestamp, I think co-pilot changed this by accident